Reply/app/src/main/java/com/materialstudies/reply/ui/MainActivity.kt

```diff
@@ -87,16 +89,16 @@ class MainActivity : AppCompatActivity(),
 
         lifecycleScope.launch {
             AdaptiveUtils.screenSizeState.collect {
+                screenSize = it
```

Comment:
Delete?

Reply:
Yeah, was using this for testing forgot to remove!